sklearn/manifold/_t_sne.py

```diff
@@ -517,13 +517,21 @@ class TSNE(BaseEstimator):
         optimization, the early exaggeration factor or the learning rate
         might be too high.
 
-    learning_rate : float, default=200.0
+    learning_rate : float or 'auto', default=200.0
         The learning rate for t-SNE is usually in the range [10.0, 1000.0]. If
         the learning rate is too high, the data may look like a 'ball' with any
         point approximately equidistant from its nearest neighbours. If the
         learning rate is too low, most points may look compressed in a dense
         cloud with few outliers. If the cost function gets stuck in a bad local
         minimum increasing the learning rate may help.
+        Note that many other t-SNE implementations (bhtsne, FIt-SNE, openTSNE,
+        etc.) use a definition of learning_rate that is 4 times smaller than
+        ours. So our learning_rate=200 corresponds to learning_rate=800 in
+        those other implementations.
+        The 'auto' option sets the learning_rate to N / early_exaggeration / 4,
+        where N is the sample size, following Belkina et al. 2019 and
+        Kobak et al. 2019, Nature Communications (or to 50.0, if
+        N / early_exaggeration / 4 < 50). This will become default in 1.2.
```

Comment:
Can we move this references into the References section below and link it here?

https://github.com/scikit-learn/scikit-learn/blob/962bd9a401bfee1f2d8e7e832018a75424b5bbe2/sklearn/manifold/_t_sne.py#L638

Reply:
Makes sense, fixed.